backend/app/db/psql_test_data.py

```diff
@@ -179,7 +181,28 @@
     CourseLessonLink(
         course_id=3,
         lesson_id=3,
+    )
+]
+
+test_users = [
+    User(
+        username="henrikpoelse@stud.kea.dk",
+        password="123",
+        full_name="John Hest",
```

Comment:
Could we have the username and `full_name` match? It would make it a lot less confusing

Also, if we could make it match the lecturer/student the id is pointing at, that would be sweet. It would make it look like we know what we are doing when demoing :p

Reply:
Username and full_name correspond now.

Not sure I understand the second part of your comment. First user has student_id 1 which is John. Second user has lecturer_ud 2 which is Peter. So they should match.

Reply:
Yy, didn't see that they already matched